spanner/tests/system/test_system.py

```diff
@@ -580,6 +581,30 @@ def test_read_w_index(self):
             [(row[0], row[2]) for row in self._row_data(ROW_COUNT)]))
         self._check_row_data(rows, expected)
 
+    def test_read_w_single_key(self):
+        ROW_COUNT = 40
+        session, committed = self._set_up_table(ROW_COUNT)
+
+        snapshot = session.snapshot(read_timestamp=committed)
+        rows = list(snapshot.read(
+            self.TABLE, self.COLUMNS, KeySet(keys=[(0,)])))
+
+        all_data_rows = list(self._row_data(ROW_COUNT))
+        expected = all_data_rows[:1]
```

Comment:
The point is that it is a *single* key (I don't want to share the implementation with the "mutiple keys" case).  I could maybe see computing `expected` via a list comprehension like:

```python
expected = [row for row in all_data_rows where row[0] == 0]
```

but I doubt it will be any clearer.